persim/pers_landscape_approx.py

```diff
@@ -0,0 +1,380 @@
+"""
+    Persistence Landscape Approximate class
+
+    authors: Gabrielle Angeloro, Michael Catanzaro
```

Comment:
How would you feel about adding your names to the `setup.py` authors list once rather than at the top of each of these files?

Reply:
Sounds good.